doc/theory/sh_basis.rst

```diff
@@ -0,0 +1,84 @@
+.. _sh-basis:
+
+========================
+Spherical Harmonic bases
+========================
+
+Spherical Harmonics (SH) are functions defined on the sphere. A collection of SH
+can used as a basis function to represent and reconstruct any function on the
+surface of a unit sphere.
+
+Spherical harmonics are ortho-normal functions defined by:
+
+..  math::
+
+    Y_l^m(\theta, \phi) = (-1)^m \sqrt{\frac{2l + 1}{4 \pi} \frac{(l - m)!}{(l + m)!}} P_l^m( cos \theta) e^{i m \phi}
+
+where $l$ is the band index, $m$ is the order, $P_l^m$ is an associated
+$l$-th degree, $m$-th order Legendre polynomial, and $(\theta, \phi)$ is the
+representation of the direction vector in the spherical coordinate.
+
+A function $f(\theta, \phi)$ can be represented using a spherical harmonics
+basis using the spherical harmonics coefficients $a_l^m$, which can be
+computed using the expression:
+
+..  math::
+
+    a_l^m = \int_S f(\theta, \phi) Y_l^m(\theta, \phi) ds
+
+Once the coefficients are computed, the function $f(\theta, \phi)$ can be
+approximately computed as:
+
+..  math::
+
+    f(\theta, \phi) = \sum_{l = 0}^{\inf} \sum_{m = -l}^{l} a^m_l Y_l^m(\theta, \phi)
+
+In HARDI diffusion imaging, the Orientation Distribution Function (ODF) is a
+function on the sphere.
+
+Several Spherical Harmonics bases have been proposed in the diffusion imaging
+literature for the computation of the ODF. DIPY implements two of these:
+
+- The basis proposed by Descoteaux *et al.* [1]_:
+
+..  math::
+
+    Y_i(\theta, \phi) =
+     \begin{cases}
+     \sqrt{2} \Re(Y_l^m(\theta, \phi)) & -l \leq m < 0, \\
+     Y_l^0(\theta, \phi) & m = 0, \\
+     \sqrt{2} \Im(Y_l^m(\theta, \phi)) & 0 < m \leq l
+     \end{cases}
+
+- The basis proposed by Tournier *et al.* [2]_:
+
+..  math::
+
+    Y_i(\theta, \phi) =
+     \begin{cases}
+     \Re(Y_l^m(\theta, \phi)) & -l \leq m < 0, \\
```

Comment:
Here in this version of the formula, we don't specify the normalization. So that's the unormalized basis, used in mrtrix 0.2. This is still what is in the code, so I'm not sure why you said that Dipy uses the "Mrtrix 3" basis now.

Reply:
True JC, my bad.